deploy/scripts/src/mobile/sts/utils/config.ts

```diff
@@ -12,5 +12,7 @@ export const config = {
   mockExternalCriBaseUrl: getEnv(`MOBILE_STS_${environment}_MOCK_EXTERNAL_CRI_BASE_URL`),
   orchestrationBaseUrl: getEnv(`MOBILE_STS_${environment}_ORCHESTRATION_BASE_URL`),
   mockClientId: getEnv(`MOBILE_STS_${environment}_MOCK_CLIENT_ID`),
-  redirectUri: getEnv(`MOBILE_STS_${environment}_REDIRECT_URI`)
+  redirectUri: getEnv(`MOBILE_STS_${environment}_REDIRECT_URI`),
+  clientInstanceKey: getEnv(`MOBILE_STS_${environment}_CLIENT_INSTANCE_KEY`),
+  sqs_queue_url: getEnv(`MOBILE_STS_${environment}_SQS_QUEUE`)
```

Comment:
Nitpick (to align with casing of other config properties above): 
```suggestion
  sqsQueueUrl: getEnv(`MOBILE_STS_${environment}_SQS_QUEUE`)
```